devsetup/standalone/openstack.sh

```diff
@@ -27,6 +27,7 @@ TLSE_ENABLED=${TLSE_ENABLED:-false}
 CLOUD_DOMAIN=${CLOUD_DOMAIN:-localdomain}
 TELEMETRY_ENABLED=${TELEMETRY_ENABLED:-true}
 OCTAVIA_ENABLED=${OCTAVIA_ENABLED:-false}
+IPA_IMAGE=${IPA_IMAGE:-"quay.io/freeipa/freeipa-server:fedora-40"}
```

Comment:
Not sure if it was tested, but why not fedora-41, so we would be fine for another year? Or even centos-9-stream

Reply:
Definitely `fedora-41` is a better choice, I agree. Not sure about `centos-9-stream`.
@marios I recall you started this, do you have an opinion?